### ISSUE TITLE
Multiline console messages are not shown in Debug Console.

### ISSUE THREAD

**Issue by MicahZoltu** [microsoft/vscode#66270]
Duplicate of #51008 which is now locked.  Was closed as fixed over 6 months ago, but the problem persists.  Either this is a regression or the fix never made it to a release.

<details><summary>Version Info</summary>

```
Version: 1.30.1 (user setup)
Commit: dea8705087adb1b5e5ae1d9123278e178656186a
Date: 2018-12-18T18:12:07.165Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
OS: Windows_NT x64 10.0.17763
```
</details>

----

### Steps to Reproduce:

1. `file.js`
```javascript
console.error(new Error('A\nB'))
```
2. `launch.json`
```json
{
    "type": "node",
    "request": "launch",
    "name": "File",
    "args": [ "${workspaceFolder}/file.js", ],
}
```
3. Start debugging via F5 or command pallet or debug panel.

### Expected Output:
```
Error: A
B
    at Object.<anonymous> (C:\Users\micah\Source\test\file.js:1:77)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
```
### Actual Output:
```
> Error: A
```

----

This would be fine if the little expansion arrow resulted in showing the rest of the error. However, the expansion arrow doesn't appear to do anything.

The bug appears to be with any object expansion as console.log({ a: 'hello', b: 'goodbye'}) also has an expansion arrow that doesn't do anything.

Does this issue occur when all extensions are disabled?: Yes

**Comment by weinand** [comment]
The issue is that an object (new Error) is passed to console.log (instead of a string).

With this we are switching the debug console from text into tree mode, so that the user can interact with the object (as long as the program is running).

Changing "file.js" to be long running:
```
setInterval(_ => {
    console.error(new Error('A\nB'));
}, 10000);
```
results in this:
![2019-01-09_11-53-48](https://user-images.githubusercontent.com/1898161/50895053-3e0de200-1405-11e9-8eb0-cb6f5eb4444e.png)


@isidorn In tree mode, we should either show multiline strings correctly for the parent node, or we should render the newlines escaped (as done in the expanded node's `message` property).

**Comment by MicahZoltu** [comment]
> as long as the program is running

This is definitely the problem.  While the sample here is significantly reduced, the problem most frequently crops up for me when my application throws an `Error` that is logged as the process exits like in this application container:

```typescript
run().then(() =>
    process.exit(0)
).catch(error => {
    console.error(error)
    process.exit(1)
});

**Comment by weinand** [comment]
If you do not want to see the error as an expandable object but just as one string, then don't use an object with the console log functions.

**Comment by MicahZoltu** [comment]
The VSCode expandable object is _super_ useful.  Right now I'm looking at the output of a script I just ran that I broke in a debugger in the middle of and being able to drill into the details and explore the object is incredibly useful.  Also, flattening the error means that my `Error` object needs a `toString` function that "does the right thing" or I need to flatten to JSON, which can be quite verbose and hard to read for some objects.

Overall, my desire is to be able to leverage the expandable-objects, but at the same time not be totally in the dark when my program exits with an expandable object in the output.

Thinking about this problem, I'm guessing that the issue is that once the program exits, the expandable object cannot be queried anymore (thus, cannot be expanded)?  Is it possible/reasonable to query one or two levels deep and cache that so that the object can be partially expanded even after process exit?

I'm looking at an object in my Debug Console right now that I expanded before the process exited.  It appears that I can expand/collapse any properties I viewed _before_ the process exited but I cannot expand any new properties.  This suggests that there _is_ some caching mechanism built in, so all that is needed is a way to auto-expand the object by default (even if it then collapses after, so it appears collapsed by default).
